profiles/common/modules/custom/og_content_type_admin/og_content_type_admin.module

```diff
@@ -1317,9 +1317,9 @@ function _og_content_type_get_all_groups($group_type = 'node') {
  * Obtain all the allowed content-types in a group retrieved.
  */
 function og_content_type_admin_access_group($bundle_name = NULL) {
-  $return = $required_site_wide = $allowed_types_group = array();
-  $gid = (int) arg(1);
-  $gid = ($gid == 0 && isset($_GET['og_group_ref'])) ? $_GET['og_group_ref'] : $gid;
+  $required_site_wide = $allowed_types_group = array();
```

Comment:
This line is not useful - see https://github.com/ec-europa/platform-dev/pull/1695/files#diff-79aa5c7d5d026589ebd4b37d69a47fa9R1330

Reply:
Fixed